packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -590,31 +594,18 @@ export class RunningSummarizer implements IDisposable {
 				this.beforeSummaryAction();
 			},
 			async () => {
+				if (this.mc.config.getBoolean("Fluid.Summarizer.TryDynamicRetries")) {
+					return this.trySummarizeWithDynamicRetries(reason, cancellationToken);
```

Comment:
I did this in a separate function to keep the PR clean and relatively easy to review.